integrations/docker/images/chip-build-tizen-qemu/Dockerfile

```
@@ -0,0 +1,212 @@
+ENV TIZEN_FS_IMAGE_DATA /opt/tizen-iot-sysdata.img
+
+WORKDIR /tmp/workdir
+
+RUN set -x \
+    # ------------------------------------------------------------------------------
+    # Download Tizen images
+    && wget --progress=dot:giga -r -nd --no-parent -A 'tizen-*.tar.gz' \
+    http://download.tizen.org/snapshots/tizen/unified/latest/images/standard/tizen-headless-armv7l/  \
+    # Unpack
+    && tar -xzf tizen-*.tar.gz \
+    && mv system-data.img $TIZEN_FS_IMAGE_DATA \
+    && mv rootfs.img $TIZEN_FS_IMAGE_ROOT \
+    # Cleanup
+    && rm -rf /tmp/workdir \
+    && : # last line
+
+WORKDIR /tmp/workdir
+
+RUN set -x \
+    # ------------------------------------------------------------------------------
+    # Install libguestfs and linux-image-generic for guestfish
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
+    libguestfs-tools \
+    linux-image-generic-hwe-20.04 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/ \
+    && SYSTEMD_SYSTEM=/usr/lib/systemd/system \
+    # Add extra libraries
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
+    $TIZEN_SDK_TOOLCHAIN/arm-tizen-linux-gnueabi/lib/libasan.so.* /usr/lib/ \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
+    $TIZEN_SDK_TOOLCHAIN/arm-tizen-linux-gnueabi/lib/libubsan.so.* /usr/lib/ \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
+    $TIZEN_SDK_SYSROOT/usr/lib/libcapi-network-thread.so.* /usr/lib/ \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
+    $TIZEN_SDK_SYSROOT/usr/lib/libnsd-dns-sd.so.* /usr/lib/ \
+    # Disable failing systemd services
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob rm \
+    $SYSTEMD_SYSTEM/deviced.service \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob rm \
+    $SYSTEMD_SYSTEM/mnt-inform.mount \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob rm \
+    $SYSTEMD_SYSTEM/murphyd.service \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob rm \
+    $SYSTEMD_SYSTEM/pulseaudio.service \
+    # Add HCI emulator (btvirt) from BlueZ project
+    && wget -O bluez.tar.gz https://github.com/bluez/bluez/archive/refs/tags/5.64.tar.gz \
+    && tar -xzf bluez.tar.gz \
+    && cd bluez-* \
+    && export CFLAGS="--sysroot=${TIZEN_SDK_SYSROOT}" \
+    && export PKG_CONFIG_PATH="${TIZEN_SDK_SYSROOT}/usr/lib/pkgconfig" \
+    && export PKG_CONFIG_SYSROOT_DIR="${TIZEN_SDK_SYSROOT}" \
+    && ./bootstrap \
+    && ./configure --host=arm-linux-gnueabi --disable-client --disable-manpages --enable-testing --enable-tools \
+    && make lib/bluetooth/bluetooth.h lib/bluetooth/hci.h \
+    && make emulator/btvirt -j$(nproc) \
+    && mkdir temp_btvirt \
+    && install emulator/btvirt temp_btvirt \
+    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
+    ./temp_btvirt/* /usr/bin/ \
```

Is it required to use `install` command before copying file to FS?

```suggestion
    && guestfish --rw -a $TIZEN_FS_IMAGE_ROOT -m /dev/sda glob copy-in \
    emulator/btvirt /usr/bin/ \
```